ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/StatusCard/StatusCard.tsx

```diff
@@ -36,6 +41,14 @@ const showFailedTestsWarning = (machine: MachineDetails) => {
   return machine.testing_status.status === TestStatusStatus.FAILED;
 };
 
+const formatSyncInterval = (syncInterval: number) =>
+  formatDuration(
+    intervalToDuration({
+      start: 0,
+      end: syncInterval * 1000,
```

Comment:
There's no indication outside of this line of code that `syncInterval` is given in seconds. What do you think about using a `type Seconds = number` alias wherever `sync_interval` is used for a bit more clarity?